infra/main.bicep

```diff
@@ -582,6 +573,17 @@ module searchReaderRoleBackend 'core/security/role.bicep' = if (useAuthenticatio
   }
 }
 
+// For computer vision access by the backend
```

Comment:
did we need to add a role for access by principal in case you are bringing an existing vision account you didn't create?